curve25519-dalek/src/specs/scalar_specs_u64.rs

```diff
@@ -46,6 +46,21 @@ pub open spec fn to_scalar(limbs: &[u64; 5]) -> nat {
     to_nat(limbs) % group_order()
 }
 
+///Converts 8 bytes into the equivalent little endian word
+pub open spec fn bytes8_to_nat(
+    b0: u8, b1: u8, b2: u8, b3: u8,
+    b4: u8, b5: u8, b6: u8, b7: u8
```

Comment:
To be more consistent across specs, I'd change this to `bytes: [u8;8]`, or `[u8;32]` and some offset var.